Data/core_rulebook/cr_abilities.lua

```diff
@@ -0,0 +1,10580 @@
+-- Converted From LST file data\pathfinder\paizo\roleplaying_game\core_rulebook\cr_abilities.lst
+-- From repository https://github.com/pcgen/pcgen at commit 11ceb52482855f2e5f0f6c108c3dc665b12af237
+SetSource({
+  SourceLong="Core Rulebook",
+  SourceShort="CR",
+  SourceWeb="http://paizo.com/store/downloads/pathfinder/pathfinderRPG/v5748btpy88yj",
+  SourceDate="2009-08",
+})
+ModifyAbility({
+  Name="Default",
```

Comment:
I assume this is a magic ability that just gets applied to everyone?

Reply:
I kinda assume so, but I'm not sure. Its just whats in the lst files.

Reply:
Aha, I found it. The original definition of the "Default" ability is in `ce_abilities_race.lst` I have no idea why its there, but there is a comment about it too.